annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -363,6 +369,7 @@ public void validateWebAnnotation(Annotation webAnnotation)
     }
 
     public void validateAnnotationId(AnnotationId annoId) throws ParamValidationException {
+    if(annoId==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.